src/lib/consent-components/components/consent-button/consent-button.component.scss

```diff
@@ -1,20 +1,20 @@
-$primary-color: var(--primary-color);
-$btn-text-color: var(--btn-text-color);
-
-.consent-btn {
-    display: block;
-    background-color: $primary-color;
-    padding: 5px 25px;
-    border-radius: 3px;
-    border: none;
-    margin: auto;
-    font-size: 14px;
-    text-decoration: none;
-    text-transform: uppercase;
-    color: $btn-text-color;
-    line-height: 33px;
-    letter-spacing: 1.25px;
-    &:hover {
-        cursor: pointer;
-    }
+$primary-color: var(--primary-color);
```

Comment:
@micpog @PiotrSzperka why are there so many white space changes. This is a new project so either provide some way of enforcing formatting in this repo to enforce new format, or don't modify whitespace.

Reply:
@rnowosielski we will enforce it in CI to make sure proper styling and linting is provided

Reply:
Fixed